platform/registry.go

```diff
@@ -91,4 +96,8 @@ func Init(platformHandles []string) {
 
 	CollectionsAPIs = getCollectionsHandlers()
 	NamingAPIs = getNamingHandlers()
+
+	compoundLendingProvider := compound.Init(viper.GetString("ethereum.compound_api"))
+	LendingAPIs = make(map[string]blockatlas.LendingAPI, 10)
```

Comment:
what is that?

Reply:
Size hint for map, to avoid multiple allocation , if there will be more.